crates/matrix-sdk-base/src/store/memory_store.rs

```diff
@@ -376,25 +377,48 @@ impl MemoryStore {
         room_id: &RoomId,
         event_type: StateEventType,
         state_key: &str,
-    ) -> Result<Option<Raw<AnySyncStateEvent>>> {
-        Ok(self
+    ) -> Result<Option<RawAnySyncOrStrippedState>> {
+        if let Some(e) = self
+            .stripped_room_state
+            .get(room_id)
+            .as_ref()
+            .and_then(|events| events.get(&event_type))
+            .and_then(|m| m.get(state_key).map(|m| m.clone()))
```

Comment:
Is `.cloned()` not applicable here because of an extra deref coercion?

Reply:
Yep, `m` is a `Ref<_, event>`.